src/NavBar1.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
```

Comment:
you copied all this from reactstrap source didn't you?

Reply:
yeah... but it wasn't as easy as copy and paste. I didn't have reactstrap or bootstrap css installed so things were breaking. Now it's great. we have  a good template.